### PR TITLE
feat: StarCanvas

### PR DESCRIPTION
**Pull request by jh3y**
## What was changed

- Fix `overflow` on body
- Sort out case studies jumping by putting logos at top and leaving the see more at the bottom. I think this feels better than both controls at the top.
- Implemented randomly generated star canvas that reacts to cursor movement
- Optional Easter Egg that I can remove. Type "temporal" and the stars will go into "Party" mode 🎉 
- We could ditch the SVG for the backdrop now, that's a ~200kb file that we don't need if we're going to use canvas and a little JavaScript 😉 

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by sw-yx** [comment]
haha nice! the stars feel a little too big compared to what we had before. bit distracting compared to the text. can we make them more subtle? like i wonder if just making them 1/3 the size would do it.

also we should probably change that coinbase copy

**Comment by sw-yx** [comment]
we could also perhaps make party mode more discoverable if you click on the top logo while on the index page...

image

any thoughts on using konami code?

**Comment by jh3y** [comment]
Yep, we can do that 👍 
I've implemented it in a way such that we change some threshold variables and play with it to see what we like the look of. Top of `StarCanvas.js` 🤓

File: components/CaseStudies.js [hunk 1]
```diff
@@ -48,9 +48,8 @@ const QUOTES = [
       <span>
         Trying to model complex SAGAS using{' '}
         <span className="text-temporalblue">a plain execution graph</span> such as Zeebe and Step
-        Functions <span className="text-temporalblue">makes things unnecessarily complex</span>,
-        plus we want a solution which we could explore for more use cases that don't necessarily use
-        SAGAS. So I think this is where Temporal really shined.
+        Functions <span className="text-temporalblue">makes things unnecessarily complex</span>. 
+        This is where Temporal really shines.
       </span>
```
Comment (sw-yx) [inline components/CaseStudies.js:53]: shortened the quote

**Comment by jh3y** [comment]
We could totally do "Konami" code 😎 

We'd need some kind of global state in place to start doing things like clicking on the logo. That might be a bit "overkill"?

**Comment by sw-yx** [comment]
re: global state - happy to just ditch the idea, was just a random suggestion

konami is more universally understood for easter eggs imo

another thing - these logos feel a little too subtle and easy to miss. perhaps a nice border to highlight them and ease the distracting background star things since we also have zoom-on-hover? side idea, what about pushing these logos to the left and spell out the company name on the right, so that its more in-your-face what company is being quoted

image

**Comment by jh3y** [comment]
Hmm. If we start taking things out of center alignment though it's going to get fiddly on smaller devices 🤔 

We could make it so the stars don't go to full opacity perhaps and start them at a lower opacity 🤔  Could apply a `backdrop-filter: blur(10px)` to the `section` but that looks kinda off in contrast to everything else.

Hmm. Could also do the section as background spaceblack with some reduced opacity perhaps 🤔  Trying to think of the easiest non-invasive solution. Unless we don't grayscale the logos?

**Comment by jh3y** [comment]
I've updated the star characteristics and it "feels" a bit less invasive. Remember it's randomly generated though so each load/resize will yield different results for people.

**Comment by jh3y** [comment]
We could also tinker with the star density and see how it looks 🤔

**Comment by sw-yx** [comment]
oh this is MUCH better

**Review by sw-yx — APPROVED** [review]
i think this PR is shippable as-is. the remaining suggestions i had were just off-the-top-of-my-head, we dont have to act on them. will leave to you on merging

**Comment by jh3y** [comment]
I'll swap out to Konami and then we're good to go 🚀

**Comment by jh3y** [comment]
Konami Code ready to go 👀 🚀

**Comment by jh3y** [comment]
p.s I don't have `write` access to merge 😄

**Comment by sw-yx** [comment]
🚢 :shipit:

**Pull request merged**